src/main/java/com/google/sps/data/DatastoreAvailabilityDao.java

```diff
@@ -147,10 +145,16 @@ public void deleteInRangeForUser(String email, long minTime, long maxTime) {
     return availability;
   }
 
-  private List<Entity> getEntitiesInRange(long minTime, long maxTime, Optional<Filter> filterOpt) {
+  private List<Entity> getEntitiesInRange(
+      Instant minTime, Instant maxTime, Optional<Filter> filterOpt) {
     Filter minFilter =
-        new FilterPredicate("startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime);
-    Filter maxFilter = new FilterPredicate("startTime", FilterOperator.LESS_THAN_OR_EQUAL, maxTime);
+        new FilterPredicate(
+            "startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime.toEpochMilli());
```

Comment:
Add a comment so that explains why startTime is used twice

Reply:
Done